### PR TITLE
MTL-2112 Install `libcsm` with `docs-csm`

### PR DESCRIPTION
**Pull request by rustydb**
# Description

This installs `libcsm` in the installation and upgrade workflows.

`libcsm` will start providing dependencies for `docs-csm`, so it should be installed at the same time.

This will need to be backported to release/1.3 and release/1.4 once reviews and feedback are incorporated.

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by heemstra — APPROVED** [review]

**Review by jacobsalmela — APPROVED** [review]
Noice

File: upgrade/Stage_0_Prerequisites.md [hunk 1]
```diff
@@ -44,47 +44,8 @@ after a break, always be sure that a typescript is running before proceeding.
...
-         > **Important:**
-         >
-         > - Replace the `PATH_TO_DOCS_RPM` below with the location of the RPM on `ncn-m001`.
-         > - The upgrade scripts expect the `docs-csm` RPM to be located at `/root/docs-csm-latest.noarch.rpm`; that is why this command copies it there.
-
-         ```bash
-         cp PATH_TO_DOCS_RPM /root/docs-csm-latest.noarch.rpm &&
-         rpm -Uvh --force /root/docs-csm-latest.noarch.rpm
-         ```
+1. (`ncn-m001#`) Install the latest `docs-csm` and `libcsm` RPMs, see the short procedure in
```
Comment (mharding-hpe) [inline upgrade/Stage_0_Prerequisites.md:47]:
```suggestion
1. (`ncn-m001#`) Install the latest `docs-csm` and `libcsm` RPMs. See the short procedure in
```

File: upgrade/Stage_2.md [hunk 2]
```diff
@@ -179,16 +179,16 @@ For any typescripts that were started earlier on `ncn-m001`, stop them with the
 
 1. (`ncn-m002#`) Copy artifacts from `ncn-m001`.
 
-   A later stage of the upgrade expects the `docs-csm` RPM to be located at `/root/docs-csm-latest.noarch.rpm` on `ncn-m002`; that is why this command copies it there.
+   > A later stage of the upgrade expects the `docs-csm` and `libcsm` RPMs to be located at `/root/` on `ncn-m002`; that is why this command copies it there.
```
Comment (mharding-hpe) [inline upgrade/Stage_2.md:182]:
```suggestion
   > A later stage of the upgrade expects the `docs-csm` and `libcsm` RPMs to be located at `/root/` on `ncn-m002`; that is why this command copies them there.
```

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**